inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/ConceptFeatureSupport.java

```diff
@@ -159,9 +156,15 @@ public String renderFeatureValue(AnnotationFeature aFeature, AnnotationFS aFs,
     @Override
     public void setFeatureValue(JCas aJcas, AnnotationFeature aFeature, int aAddress, Object aValue)
     {
-        KBHandle kbInst = (KBHandle) aValue;
         FeatureStructure fs = selectByAddr(aJcas, FeatureStructure.class, aAddress);
-        setFeature(fs, aFeature, kbInst != null ? kbInst.getIdentifier() : null);
+        String kbIdentifier;
+        if (aValue instanceof KBHandle) {
```

Comment:
Please commend here in which cases the value is a KBHandle and in which cases it is a String. It is unclear how this could happen.